Dockerfile

```
@@ -80,7 +80,10 @@ RUN python3 -m pip install --no-deps \
         pytijo==0.0.2 \
         pyzbar-upright==0.1.8 \
         rawpy==0.19.1 \
-        scipy==1.11.4
+        scipy==1.11.4 \
+        numpy==1.26.4 \
+        pillow 
```

Also force a version here:
```suggestion
        pillow==10.3.0 
```